charts/tidb-drainer/values.yaml

```diff
@@ -48,6 +48,22 @@ tlsCluster:
   certAllowedCN: []
   #  - TiDB
 
+# The tls config between drainer and the downstream database server (MySQL/TiDB)
+tlsSyncer:
+  tlsClientSecretName:
+
+  # certAllowedCN is the Common Name that allowed
+  certAllowedCN: []
+  #  - TiDB
+
+  # checkpoint is the tls config for the database we save binlog checkpoint
+  # Omit this part if you just want to save checkpoint in downstream database
+  checkpoint:
+    tlsClientSecretName:
```

Comment:
Add an explanation, and if the DB of saving checkpoint is the same as the downstream DB, this can use the same secret as in L53, right? If yes, make it clear in the comments.
And, in this case, if users do not configure the `checkpoint`, we should use the configuration in L53-L56?

Reply:
No, empty configuration is okay. binlog will fill the security options automatically if `[syncer.to.checkpoint.type]` if not specified.